.gitignore

```diff
@@ -2,3 +2,4 @@
 src/.DS_Store
 .DS_Store
 /.idea/
+/.vscode/
```

Comment:
Some of the local settings somehow sneaked into our .gitignore, but lets not make it a rule :) 

I would prefer to keep `*~`, `*.bak`, `thumbs.db` or other similar files like editor settings in a global .gitignore, https://stackoverflow.com/questions/7335420/global-git-ignore

Reply:
Whoops. Fixed that.

Reply:
👍